libbpf-rs/src/program.rs

```diff
@@ -73,6 +73,7 @@ pub enum ProgramType {
     StructOps,
     Ext,
     Lsm,
+    SkLookup,
```

Comment:
same here, we now also have BPF_PROG_TYPE_SYSCALL, would be good to add

Reply:
Sounds good. Done!